package.json

```
@@ -59,7 +59,7 @@
     "eslint-import-resolver-typescript": "2.0.0",
     "eslint-plugin-chai-friendly": "0.5.0",
     "eslint-plugin-import": "^2.22.0",
-    "ethers": "4.0.45",
+    "ethers": "5.3.1",
```

Should use latest version. It's probably been released since you started working on it 😆 . Applies to all of the `package.json` files.
```suggestion
    "ethers": "5.4.0",
```

---

In my opinion, I prefer having the exact version